packages/shared/src/components/Feed.tsx

```diff
@@ -148,8 +156,13 @@ export default function Feed<T>({
     !user?.acquisitionChannel;
   const { getMarketingCta } = useBoot();
   const marketingCta = getMarketingCta(MarketingCtaVariant.Card);
+  const plusEntry = getMarketingCta(MarketingCtaVariant.PlusCard);
+  const isMobile = useViewSize(ViewSize.MobileXL);
+  const { value: plusEntryMobile } = useConditionalFeature({
+    feature: featurePlusEntryMobile,
+    shouldEvaluate: isMobile && !!plusEntry,
+  });
```

Comment:
Same comment goes here. Maybe it would be better to contain the conditions in a hook.